src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -863,9 +887,13 @@ namespace wallet_rpc
     struct request_t
     {
       std::string address;    // Destination public address.
-      uint32_t priority;      // (Optional) Priority for sending the sweep transfer, partially determines fee.
+      uint32_t priority;      // Set a priority for the transaction. Accepted Values are: 0-5 for: default, unimportant, normal, elevated, priority, blink.
       bool blink;             // Specifies that the tx should be blinked (`priority` will be ignored).
+#if LOKI_DEPRECATE_AT_VERSION(7, 1, 0)
       uint64_t ring_size;     // (Deprecated) Set to 10. Sets ringsize to n (mixin + 1). Loki ring_size is statically set to 10.
```

Comment:
This seems out of place?  Both blink/ring_size should be in the future-deprecated region?

Also: as above, I think we can just drop both.